core/actions/datacite/action.ts

```diff
@@ -9,9 +9,9 @@ export const action = defineAction({
 	name: Action.datacite,
 	config: {
 		schema: z.object({
-			doi: z.string(),
+			doi: z.string().optional(),
 			doiPrefix: z.string().optional(),
-			doiSuffix: z.string(),
+			doiSuffix: z.string().optional(),
```

Comment:
Snuck in a small configuration fix for the DataCite action. Because you no longer need to specify a fallback DOI (or suffix), we can now take advantage of auto-generated DataCite DOIs.